core/src/consensus/tendermint/stake/action_data.rs

```diff
@@ -29,7 +29,7 @@ fn get_account_key(address: &Address) -> H256 {
 }
 
 lazy_static! {
-    pub static ref stakeholder_addresses_key: H256 =
+    pub static ref STAKEHOLDER_ADDRESSES_KEY: H256 =
```

Comment:
Make another PR for it.
Please don't mix up a commit for refactoring/clean up code with a patch feature implementation.

Reply:
Okay